src/core_plugins/kibana/server/tutorials/netflow/on_prem_elastic_cloud.js

```diff
@@ -18,40 +18,47 @@
  */
 
 import { INSTRUCTION_VARIANT } from '../../../common/tutorials/instruction_variant';
-import { LOGSTASH_INSTRUCTIONS } from '../../../common/tutorials/logstash_instructions';
+import { createLogstashInstructions } from '../../../common/tutorials/logstash_instructions';
 import {
-  TRYCLOUD_OPTION1,
-  TRYCLOUD_OPTION2
+  createTrycloudOption1,
+  createTrycloudOption2,
 } from '../../../common/tutorials/onprem_cloud_instructions';
-import { COMMON_NETFLOW_INSTRUCTIONS } from './common_instructions';
+import { createCommonNetflowInstructions } from './common_instructions';
 
 // TODO: compare with onPrem and elasticCloud scenarios and extract out common bits
-export const ON_PREM_ELASTIC_CLOUD_INSTRUCTIONS = {
-  instructionSets: [
-    {
-      title: 'Getting Started',
-      instructionVariants: [
-        {
-          id: INSTRUCTION_VARIANT.OSX,
-          instructions: [
-            TRYCLOUD_OPTION1,
-            TRYCLOUD_OPTION2,
-            ...LOGSTASH_INSTRUCTIONS.INSTALL.OSX,
-            ...COMMON_NETFLOW_INSTRUCTIONS.CONFIG.ON_PREM_ELASTIC_CLOUD.OSX,
-            ...COMMON_NETFLOW_INSTRUCTIONS.SETUP.OSX
-          ]
-        },
-        {
-          id: INSTRUCTION_VARIANT.WINDOWS,
-          instructions: [
-            TRYCLOUD_OPTION1,
-            TRYCLOUD_OPTION2,
-            ...LOGSTASH_INSTRUCTIONS.INSTALL.WINDOWS,
-            ...COMMON_NETFLOW_INSTRUCTIONS.CONFIG.ON_PREM_ELASTIC_CLOUD.WINDOWS,
-            ...COMMON_NETFLOW_INSTRUCTIONS.SETUP.WINDOWS
-          ]
-        }
-      ]
-    }
-  ]
-};
+export function createOnPremElasticCloudInstructions() {
```

Comment:
There'll be much fewer diffs in PRs like this if the export was like:
```js
const COMMON_NETFLOW_INSTRUCTIONS = createCommonNetflowInstructions();
const TRYCLOUD_OPTION1 = createTrycloudOption1();
const TRYCLOUD_OPTION2 = createTrycloudOption2();
const LOGSTASH_INSTRUCTIONS = createLogstashInstructions();

export const createOnPremElasticCloudInstructions = () => ({
    instructionSets: [
      //...
    ]
});
```

that is, using implicit return should keep the indentation the same

Reply:
@tsullivan 
`createCommonNetflowInstructions()`, ..., `createLogstashInstructions()` can't be called outside the function, because i18n engine should be initialized before text translation.